src/main/java/frc/robot/subsystems/shooter/Shooter.kt

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.shooter
+
+import edu.wpi.first.wpilibj2.command.Subsystem
+import frc.robot.CANDevice
+import org.littletonrobotics.junction.Logger
+
+
+object Shooter : Subsystem {
+
+    // TODO: Implement feedforward maybe
+
+
+    private val inputs = ShooterIO.ShooterInputs()
+
+    private val io = ShooterIOReal(CANDevice.FlywheelMotor.id, CANDevice.ShooterFeedMotor.id)
+
+
+    override fun periodic() {
+        io.updateInputs(inputs)
+
+        Logger.getInstance().recordOutput("Shooter/YeetMainMotorVelocity", io.getSpeedFlywheel())
+        Logger.getInstance().recordOutput("Shooter/YeetSecondaryMotorVelocity", io.getSpeedFeeder())
```

Comment:
This is _not_ how AdvantageKit works. These should be recorded in `IntakeIO.Inputs`. Also, the inputs need to be processed by `Logger`.